k-distribution/tests/regression-new/profile/Makefile

```diff
@@ -1,5 +1,11 @@
 DEF=test
 EXT=test
-KOMPILE_FLAGS=--profile-rule-parsing --syntax-module TEST
+KOMPILE_BACKEND=java
```

Comment:
Why was the backend of this test changed? In general we want to avoid adding new tests that run on the ocaml or java backends when possible, unless we are specifically testing those backends.

Reply:
That's fair. I needed to test kprove besides kompile. I will change it to haskell.